laos_gggi/co2_processing.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+import os
+
+
+def process_co2():
+    if not os.path.isfile("../data/co2.csv"):
+        url = "https://gml.noaa.gov/aftp/products/trends/co2/co2_mm_gl.csv"
```

Comment:
move this URL to the constants file and import it into this file. Check the other data downloaders for the general pattern to follow